Core/Graphics/Source/GraphicsImpl.cpp

```diff
@@ -37,23 +38,25 @@ namespace Babylon
         DisableRendering();
     }
 
-    void* Graphics::Impl::GetNativeWindow()
+    WindowType Graphics::Impl::GetNativeWindow()
     {
         std::scoped_lock lock{m_state.Mutex};
-        return m_state.Bgfx.InitState.platformData.nwh;
+        return static_cast<WindowType>(m_state.Bgfx.InitState.platformData.nwh);
     }
 
-    void Graphics::Impl::SetNativeWindow(void* nativeWindowPtr, void* windowTypePtr)
+    void Graphics::Impl::SetNativeWindowInternal(GraphicsConfiguration config)
     {
         std::scoped_lock lock{m_state.Mutex};
         m_state.Bgfx.Dirty = true;
 
         auto& pd = m_state.Bgfx.InitState.platformData;
-        pd.ndt = windowTypePtr;
-        pd.nwh = nativeWindowPtr;
+        pd.ndt = nullptr;
+        pd.nwh = config.windowPtr;
```

Comment:
UWP is the only platform that makes use of the platformData.ndt to specify the type of window being drawn to, so it was added as a field to the configuration object used to create the Graphics object now. 

This should be easily extensible to include any other datatypes needed for UWP as seen in #606.

Reply:
Following @ryantrem, suggestion I removed `Graphics::Impl::SetNativeWindowInternal` in favor of having platform implementations of `Graphics::Impl::ConfigureBgfxPlatformData` called by a shared `Graphics::Impl::SetNativeWindow`